app/src/main/java/org/wikipedia/notifications/NotificationActivity.kt

```diff
@@ -288,7 +287,20 @@ class NotificationActivity : BaseActivity() {
         }
         val checkExcludedWikiCodes = NotificationsFilterActivity.allWikisList().size != includedWikiCodes.size
 
+        var allUnreadCount = 0
+        var mentionsUnreadCount = 0
+
         for (n in filteredList) {
+            val isMention = NotificationCategory.isMentionsGroup(n.category)
+            if (n.isUnread) {
+                allUnreadCount++
+                if (isMention) {
+                    mentionsUnreadCount++
+                }
+            }
```

Comment:
They will not be counted correctly if the `excludedTypeCodes` is not empty. Should we move the logic after line `315` or some other place?

Reply:
Whoops, that's correct! How about now?